### ISSUE TITLE
Can't locate an element by class with Watir 6.12

### ISSUE THREAD

**Issue by sandeepnagra** [watir/watir#776]
## Meta -
Watir Version: 6.12
Selenium Version: 3.14.0
Browser Version: Chrome 69
Browser Driver Version: 2.35
OS Version: OSX 10.13 (High Sierra)

## Expected Behavior - Following script should work with Watir 6.12 as it works fine in Watir 6.10.1
```
require 'watir'
browser = Watir::Browser.new :chrome
browser.goto 'https://advisors.massmutual.com/'
fail 'Missing state plot on map' unless browser.path(class: 'massachusetts').present?
browser.close 
```

## Actual Behavior - Above script fails when I am using class to locate the element after upgrading to Watir 6.12 but if use css with same class name it works like shown below.
```
require 'watir'
browser = Watir::Browser.new :chrome
browser.goto 'https://advisors.massmutual.com/'
fail 'Missing state plot on map' unless browser.path(css: '.massachusetts').present?
browser.close
```
## Steps to reproduce -
Above script should be enough to reproduce the issue.

**Comment by titusfortner** [comment]
Huh, this looks like a straight up XPath issue.

This should work, but doesn't:
```ruby
.//path[contains(concat(' ', @class, ' '), ' feature ')
```
This does work for some reason:
```ruby
.//*[contains(concat(' ', @class, ' '), ' feature ')]
```

So a temp fix is to use:
```ruby
browser.element(class: 'massachusetts').present?
```

@lmtierney / @p0deje / @jkotests - Anyone have any insight into why this might be happening?

**Comment by twalpole** [comment]
This is probably due to the default namespace of svg elements in HTML5 which XPath cares about but CSS  doesn't - One fix could be to use `*[local-name()='path']' in the XPath expression.

**Comment by sandeepnagra** [comment]
@titusfortner - I have tried `browser.element(class: 'massachusetts').present?` as well but it didn't work either.

**Comment by titusfortner** [comment]
@sandeepnagra 
It's working for me:

```
irb(main):017:0> browser.element(class: 'massachusetts').exists?
2018-09-13 13:36:55 INFO Selenium -> POST session/673cf8f5902a8f63d4e17e8f14ddb112/element
2018-09-13 13:36:55 INFO Selenium    >>> http://127.0.0.1:9515/session/673cf8f5902a8f63d4e17e8f14ddb112/element | {"using":"class name","value":"massachusetts"}
2018-09-13 13:36:55 INFO Selenium <- {"sessionId":"673cf8f5902a8f63d4e17e8f14ddb112","status":0,"value":{"ELEMENT":"0.4163224081326158-2"}}
=> true
irb(main):018:0> browser.element(class: 'massachusetts').present?
2018-09-13 13:37:04 INFO Selenium -> POST session/673cf8f5902a8f63d4e17e8f14ddb112/element
2018-09-13 13:37:04 INFO Selenium    >>> http://127.0.0.1:9515/session/673cf8f5902a8f63d4e17e8f14ddb112/element | {"using":"class name","value":"massachusetts"}
2018-09-13 13:37:04 INFO Selenium <- {"sessionId":"673cf8f5902a8f63d4e17e8f14ddb112","status":0,"value":{"ELEMENT":"0.4163224081326158-2"}}
2018-09-13 13:37:04 INFO Selenium -> GET session/673cf8f5902a8f63d4e17e8f14ddb112/element/0.4163224081326158-2/displayed
2018-09-13 13:37:05 INFO Selenium <- {"sessionId":"673cf8f5902a8f63d4e17e8f14ddb112","status":0,"value":true}
=> true
```

**Comment by lmtierney** [comment]
@titusfortner isn't the similar to the issue we saw in wtr-general with the person automating Firefox's internal page? And I assumed it was just the xpath on that internal page, because similar xpath worked on a google page

**Comment by titusfortner** [comment]
@lmtierney do you remember which was the page? I can verify with the new code.

**Comment by lmtierney** [comment]
@titusfortner it was `about:support`. On another note, @twalpole 's suggestion seems to work

**Comment by titusfortner** [comment]
Yeah, #777 is the fix that should go out with 6.14

**Comment by sandeepnagra** [comment]
Thank you all! I verified that this issue is fixed in version 6.14
